RELEASE/scripts/autoscend/combat/auto_combat_default_stage2.ash

```diff
@@ -159,6 +159,12 @@ string auto_combatDefaultStage2(int round, monster enemy, string text)
 		//TODO use $item[Tattered Scrap Of Paper] as a free runaway?
 		//TODO use $item[Green Smoke Bomb] as a free runaway?
 
+		string freeRunSource = findFreeRunSource(true);
```

Comment:
this should go after all the banishes, in case you get into a situation where a banisher is prepared but the navel ring is also equipped. Banishes should have higher priority than free runs

Reply:
Oh my bad, the banishers are already before it's just peel out below this